src/platform.rs

```diff
@@ -511,10 +663,12 @@ mod tests {
         vp.set_registers(&reg_names, &reg_values).unwrap();
         vp.get_registers(&reg_names, &mut reg_values_out).unwrap();
 
+        println!("\nreg_values_out: {:?}", unsafe { reg_values_out[0].Reg64 });
```

Comment:
For some reason, WHvGetVirtualProcessorRegisters crashes if we include this line: https://pastebin.com/raw/k7jgPzq0
Apparently, we're already skipping a similar test: eb51d827cb97ebddfc3e77d4b741030838256d2b

Reply:
Yes this is quite strange, for me the test passes locally (though it doesn't when the print line is removed. Not that it's a fix, but clearly there's some non-determinism that needs to be investigated). And though I have that commit in my branch, I don't see where/when that ignore line got removed. I'll add it back (removing the extraneous print) to maintain the intention stated in that commit to investigate it further.

Reply:
Yes, this is a strange test failure. Looks like in the pastebin snippet above that it is failing at the get_registers() call. Locally my test doesn't fail with the print statement, but does fail without it (not saying that it's a valid fix, but just pointing out that there's clearly some strange non-determinism causing the failure). I'll annotate it with the same ignore notation as the other test, to maintain the intention to investigate it further.